Modules/Filtering/FFT/include/itkFFTWCommon.h

```diff
@@ -228,15 +226,13 @@ class Proxy<float>
                PixelType *   in,
                ComplexType * out,
                unsigned int  flags,
-               int           threads = 1,
+               int           itkNotUsed(threads) = 1,
                bool          canDestroyInput = false)
   {
     //
 #  ifndef ITK_USE_CUFFTW
     const std::lock_guard<FFTWGlobalConfiguration::MutexType> lockGuard(FFTWGlobalConfiguration::GetLockMutex());
     fftwf_plan_with_nthreads(threads);
```

Comment:
Requested changes (2) because the `threads` parameter is actually used here! Please use `[[maybe_unused]]` in this case!